test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -1428,3 +1428,89 @@
       initNodes: 1000
       deletingPods: 1000
       measurePods: 1000
+
+# This test case is used to measure queueing hints performance when handling NodeAdd event.
+- name: EventHandlingNodeAdd
+  featureGates:
+    SchedulerQueueingHints: true
+  workloadTemplate:
+  # Collect metrics from all createPods ops.
+  - opcode: startCollectingMetrics
+    name: unschedPods
+    namespaces: [namespace-2, namespace-5, namespace-9, namespace-10, namespace-11, namespace-15]
+  # Create one unschedulable node.
+  - opcode: createNodes
+    count: 1
+    nodeTemplatePath: config/event_handling/add-node-unschedulable.yaml
+  # Created pods blocked using NodeUnschedulable plugin.
+  - opcode: createPods
+    countParam: $measurePods
+    podTemplatePath: config/event_handling/add-node-unsched-pod-nodeunschedulable.yaml
+    skipWaitToCompletion: true
+  # Wait for pods to be processed by the scheduler.
+  - opcode: sleep
```

Comment:
I don't like the idea of using a plain sleep to wait for pods to be processed. Could we have an operation for "wait for pods to be processed" or a configuration for "barrier" to look at this instead of scheduled pods? Maybe we can be using the schedule_attempts_total metric?

Reply:
Created #127700

Reply:
Please rebase once that PR is merged

Reply:
Done